homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -176,6 +178,14 @@ def media_image_url(self):
         """Image url of current playing media."""
         return self._media_image_url
 
+    @property
+    def volume_level(self):
+        """Volume level of the media player (0..1)."""
+        if self._volume is None:
+            return None
+
+        return float(self._volume) / (self._volume_steps - 1)
```

Comment:
Please create a separate variable `self._volume_level` and move the calculation to `async_update`

Reply:
Thanks, I've made the suggested changes in the latest commit.